src/assets/cipher-card.css

```
@@ -271,3 +271,47 @@
+.anatomy-card {
+  background: rgba(0, 0, 0, 0.3);
+  border: 1px solid var(--cryptotron-border-glow);
+  padding: 1.25rem;
+  border-radius: 8px;
+  text-align: center;
+}
+
+.anatomy-card h4 {
+  color: var(--neon-green);
+  margin-bottom: 0.75rem;
+  font-family: 'Orbitron', sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)